File: stadiljus.js
// Compact version for copy-paste into online judges
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let lines = [];
rl.on('line', (line) => {
    lines.push(line);
    if (lines.length === 4) {
        const n = parseInt(lines[0]);
        const k = parseInt(lines[1]);
        const m = parseInt(lines[2]);
        const lengths = lines[3].split(' ').map(Number);
        
        lengths.sort((a, b) => a - b);
        
        let totalLength = 0;
        let count = 0;
        
        for (let length of lengths) {
            if (k * (totalLength + length) <= m * (count + 1)) {
                totalLength += length;
                count++;
            }
        }
        
        console.log(count);
        rl.close();
    }
});

File: stadiljus.py

# Read inputs
n = int(input())
k = int(input())
m = int(input())

# Read the list of lengths
lengths = list(map(int, input().split()))

# Sort the lengths in increasing order
lengths.sort()

total_length = 0
count = 0

# Go through each length
for length in lengths:
    # Check if we can include this length
    if k * (total_length + length) <= m * (count + 1):
        total_length += length
        count += 1

# Print the result
print(count)
